rllib/models/tests/test_base_model.py

```diff
@@ -118,7 +120,7 @@ def test_initial_state(self):
         and outputs correct values."""
         output = NpRecurrentModelImpl().initial_state()
         desired = TensorDict({"in": np.arange(1)})
-        for k in output.flatten().keys() | desired.flatten().keys():
+        for k in output.shallow_keys() | desired.shallow_keys():
```

Comment:
You actually want the flattened keys which means you have to call `output.keys()`?